### DISCUSSION TITLE
Can Damask get grain boundaries during post processing？

### DISCUSSION THREAD

**Discussion by zzzyhp** [damask-multiphysics/DAMASK#47] [Q&A]
Hi all,
The stress and strain nephogram obtained by post processing with Damask has no obvious grain boundary. Is it possible to get grain boundaries during posts processing to obtain the stress nephogram with grain boundaries?

**Comment by MarDiehl** [comment]
It exists for grid files, i.e. for the undeformed configuration https://damask-multiphysics.org/documentation/processing_tools/pre-processing.html#damask.GeomGrid.get_grain_boundaries
